it/mail/chrome/messenger/messenger.properties

```diff
@@ -1102,16 +1102,16 @@ hide=hide
 
 firstNameRegexp=[^\\s,]*,\\s*(.*)|([^\\s]*)\\s*.*
 kiloByteAbbreviation=KB
-pending=Pending
-templatePlaceholdersWarning=Press "tab" to quickly move between the blanks - highlighting will not be visible to recipients.
-stillContainsPlaceholdersConfirmTitle=Placeholders Detected
-byteAbbreviation=bytes
+pending=In attesa
+templatePlaceholdersWarning=Premi il tasto tabulazione per muoverti velocemente tra i campi - l'evidenziatura non sarà visibile per i destinatari.
```

Comment:
CORRECTION NEEDED. 

I would shorten the sentence by using "TAB" instead of "tabulazione".